CMakeLists.txt

```diff
@@ -407,18 +407,56 @@ add_library(onnxifi_loader STATIC onnx/onnxifi_loader.c)
 set_target_properties(onnxifi_loader PROPERTIES
   C_STANDARD 90
   C_EXTENSIONS NO)
-target_link_libraries(onnxifi_loader PUBLIC onnxifi)
+target_link_libraries(onnxifi_loader PUBLIC onnxifi ${CMAKE_DL_LIBS})
+if(MSVC)
```

Comment:
(In a follow up PR) Please create a cmake function to avoid repeating this chunk